tests/e2e/models/mnist_train_test/mnist_train_test.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from iree.runtime import load_vm_flatbuffer_file
+from iree.compiler.tools import compile_file, InputType
+import os
+import sys
+import tempfile
+import numpy as np
+import unittest
+from iree.testing import assert_array_list_almost_equal, assert_array_almost_equal, parse_args
+
+args = None
+
+
+def build_module():
+  with tempfile.TemporaryDirectory() as tmp_dir:
+    vmfb_file = os.path.join(tmp_dir, "mnist_train.vmfb")
+    compile_file(input_file=os.path.join(os.path.dirname(__file__),
+                                         "mnist_train.mlir"),
```

Comment:
Please include instructions on how this file was generated and how to regenerate it if necessary

Reply:
I added a script to generate both the model and the test data. I added instruction to the test docstring.

Reply:
Can you please add a README and/or doc string. Doesn't need to be long. I will augment for instructions on how to upload to GCS

Reply:
I added a `README.md`.